docs/walkthrough.md

```diff
@@ -138,7 +138,16 @@ chmod +x ./kubectl
 We'll assume hereafter that all `kubectl` commands are using this
 newly-installed executable.
 
-To configure `kubectl` to communicate with the service catalog APU server:
+## Step 5 - Configuring `kubectl` to Talk to the API Server
+
+To configure `kubectl` to communicate with the service catalog API server, we'll have to
+get the IP address of the `Service` (and just the IP - no ports or `http://` prefixes)
```

Comment:
alternatively, which sort of goes back to the original you had, "ports or protocols, for example, an `http(S):// prefix`. 

I think is fine as it is.